docs/conf/vilnius/2019/tickets.rst

```diff
@@ -0,0 +1,98 @@
+:template: {{year}}/generic.html
+
+
+Tickets
+=======
+
+{% if flagticketsonsale %}
+
+**Tickets are on sale now!**
+
+{% else %}
+
+**Tickets are sold out!**
+
+{% endif %}
+
+Each ticket includes:
+
+* Entry to all conference events and activities
+* Breakfast, snacks, and lunch on all conference days
+* Welcome Reception and Social Event with light snacks and free drinks
```

Comment:
maybe remove reception and social event until we know for sure whether they're happening